packages/core/src/external/carequality/ihe-gateway-v2/schema.ts

```diff
@@ -9,9 +9,10 @@ export const schemaOrArrayOrEmpty = <T extends z.ZodTypeAny>(schema: T) =>
 export const textSchema = z.union([
   z.string(),
   z.object({
-    _text: z.string(),
+    _text: z.union([z.string(), z.number()]),
```

Comment:
` {
                                    "code": "invalid_type",
                                    "expected": "string",
                                    "received": "number",
                                    "path": [
                                        "Envelope",
                                        "Header",
                                        "Security",
                                        "Assertion",
                                        "AttributeStatement",
                                        "Attribute",
                                        7,
                                        "AttributeValue"
                                    ],
                                    "message": "Expected string, received number"
                                }` 
                             we got this error in prod